tools/build-overrides/build-overrides.sh

```diff
@@ -425,8 +418,7 @@ cat <<EOF > "${decisioncentral_overrides_json}"
 {
   "artifacts": [
     {
-      "name": "DECISION_CENTRAL_DISTRIBUTION_ZIP",
-      "target": "decision_central_distribution.zip",
+      "name": "rhpam_decision_central_distribution.zip",
```

Comment:
Decision Central needs to have prefix rhdm
```suggestion
      "name": "rhdm_decision_central_distribution.zip",
```